lib/jsdom/virtual-console.js

```diff
@@ -17,9 +17,10 @@ module.exports = class VirtualConsole extends EventEmitter {
 
     for (const method of Object.keys(anyConsole)) {
       if (typeof anyConsole[method] === "function") {
-        this.on(method, () => {
```

Comment:
Because arguments[0] === anyConsole and arguments[1] === options when used in arrow function. This means that with arrow function, the console will always output itself and options, unless I miss something.